package.json

```diff
@@ -7,11 +7,15 @@
   "schematics": "./schematics.json",
   "scripts": {
     "prepublishOnly": "npm run build && npm run test",
-    "build": "npm run clean && tsc -p tsconfig.node.json && tsc -p tsconfig.browser.json && npm run :copy-schematic-files",
-    "clean": "rimraf lib",
+    "build": "npm run clean && npm run node && npm run parcel && npm run spa && npm run :copy-schematic-files",
+    "clean": "rimraf lib parcel",
     "test": "jest",
     "lint": "eslint ./src/**/*.ts",
-    ":copy-schematic-files": "cpx \"src/**/_files/**/**\" lib"
+    ":copy-schematic-files": "cpx \"src/**/_files/**/**\" lib",
+    "packagr": "ng-packagr -p ./ng-package.json",
```

Comment:
Since this is not part of the `build` script anymore, it won't be run before the library is published to npm. Does it need to be run before publishing this to npm?

If not, then can we delete ng-packagr entirely?

Reply:
True, I will remove it